action.yml

```diff
@@ -0,0 +1,51 @@
+name: 'Leviathan Action'
+description: 'Test your software directly on hardware using the Levaithan Testing framework'
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: (Virtual) Leviathan build & test 
+      working-directory: ${{ env.LEVIATHAN_ROOT }}/
+      if: env.WORKER_TYPE == 'qemu'
+      shell: bash
+      run: |
+        make config
+        make build
+        make local-test QEMU_SECUREBOOT=${QEMU_SECUREBOOT} FLASHER_SECUREBOOT=${FLASHER_SECUREBOOT} || exit 1
+
+    - name: (Autokit) Leviathan build & test
+      working-directory: ${{ env.LEVIATHAN_ROOT }}/
+      if: env.WORKER_TYPE == 'testbot' || env.WORKER_TYPE == 'autokit'
+      shell: bash
+      run: |
+        make config
+        make build
+        make test || exit 1
```

Comment:
```suggestion
        make test
```
exit 1 is implied

Reply:
Copied from Jenkins in the hopes for absolutely making sure it's a successful job and not a false positive.

Reply:
still no need